### PR TITLE
Shell check fixes

### PR DESCRIPTION
**Pull request by SamBarker**
### Type of change

_Select the type of your PR_

- Bugfix
- Enhancement / new feature
- Refactoring
- Documentation

### Description

_Please describe your pull request_

### Additional Context

_Why are you making this pull request?_

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Review performance test results. Ensure that any degradations to performance numbers are understood and justified.
- [ ] Make sure all Sonar-Lint warnings are addressed or are justifiably ignored.
- [ ] Update documentation
- [ ] Reference relevant issue(s) and close them after merging
- [ ] For user facing changes, update CHANGELOG.md (remember to include changes affecting the API of the test artefacts too).

### CODE REVIEW

**Review by k-wall — APPROVED** [review]
Not tested but lgtm

**Comment by SamBarker** [comment]
[Example results](https://github.com/SamBarker/kroxylicious/pull/6/files) for a deliberately broken branch. I quite like the way it directly annotates the PR, but might need to increase the privileges used for the token used for the PR run.

File: .github/workflows/static-analysis.yaml [hunk 1]
```diff
@@ -1,11 +1,63 @@
-name: Sonar Scan
+name: Static Analysis
```
Comment (SamBarker) [inline .github/workflows/static-analysis.yaml:1]: Most of this changes to this file go away as I've moved shell check back to the maven build (so it runs in the context of the PR and not on main with access to the Sonar token)

**Review by SamBarker — COMMENTED** [review]
It might also be worth looking at other [Reviewdog actions](https://github.com/reviewdog), such as `yamllint`

**Pull request merged**